srtcore/buffer.cpp

```diff
@@ -240,7 +240,11 @@ void CSndBuffer::updInputRate(uint64_t time, int pkts, int bytes)
       m_iInRateBytesCount += bytes;
       if ((time - m_InRateStartTime) > m_InRatePeriod) {
          //Payload average size
-         m_iAvgPayloadSz = m_iInRateBytesCount / m_iInRatePktsCount;
+         if (m_iInRatePktsCount == 0) {
+             m_iAvgPayloadSz = 0;
```

Comment:
Not sure about that. Shouldn't this take the value from `SRTO_PAYLOADSIZE` (or chose the value of `SRT_LIVE_MAX_PLSIZE` if the latter is zero)? If this value is used in calculations, by setting zero here you risk another division by zero in another place.

Reply:
After a deeper review it appears that this CSndBuffer member, not only is not used as a divider, but is not used at all. Its equivalent in the CRcvBuffer has some use and CSndBuffer.m_iAvgPayloadSz  was probably implemented for symmetry. It is safe and not too much invasive to just remove it.

Reply:
`LiveCC` has its own `m_zSndAvgPayloadSize`, therefore this value is not used.